app/views/teams/index.html.slim

```diff
@@ -2,7 +2,7 @@ nav.actions
   ul.list-inline
     li = link_to 'My Team', current_student.current_team, class: 'btn btn-default btn-sm' if current_student.current_team
     li = link_to icon('plus', 'New Team'), new_team_path, class: 'btn btn-primary btn-sm' if can? :create, Team.new(season: Season.current)
-  div.pull-right.dropdown
+  div.pull-right.dropdown.center-dropdown
     button.dropdown-toggle.form-control  data-toggle="dropdown"
```

Comment:
I think one of the problems with this button is that it's just using the _"wrong"_ css classes - one that's normally intended for forms. Maybe have a look at [the old bootstrap docs](https://bootstrapdocs.com/v3.3.5/docs/components/#btn-dropdowns-single) what markup they use in their examples.

I could imagine something like:
```ruby
button.dropdown-toggle.btn.btn-default data-toggle='dropdown'
```
to work with the right colors etc.

Do you wanna give it a try? Maybe better on the long run compared to overriding bootstrap defaults 🤔